.eslintrc.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path')
```

Comment:
I moved eslint config to here to use the `path.resolve` in `parserOptions.project`